### ISSUE TITLE
Fix #1366 should be replicated for template parsing when sending emails via workflow

### ISSUE THREAD

**Issue by LuminisIndia** [salesagility/SuiteCRM#2659]
I followed the solution proposed at: [Fixed #1366 - Using parse_template_with_bean instead of parse_template_bean #2562](https://github.com/salesagility/SuiteCRM/pull/2562/commits/d9d213b2cf7133562d172301cc6707aff598fcf1) by 	[daniel-samson](https://github.com/daniel-samson)

#### Issue
Though I am not sure because I am not a coder/expert about this, I guess the above solution is 100% working.  As we have followed this and solution worked absolutely fine with normal email sending via Activities Menu. Since the success of above solution was noteworthy, I wish to have the same template parsing approach for tempateParsing within AOW actions too.

#### Expected Behavior
Users would be delighted to have same accuracy of template parsing when sending emails via AOW actions.

#### Actual Behavior
However, when I tried to use the same template for sending emails via workflow, AOW failed to convert the datetime fields correctly. 

#### Steps to Reproduce

1. Please refer to discussion at URL:
https://www.suitecrm.com/forum/advanced-opensales-workflow-portal-support/11983-emails-sent-via-workflow-showing-only-date-but-not-time-of-meeting#40670
2.
3.
4.

#### Context
I think it would bring a huge consistency to emails sending via workflows.  Because almost 100% of users of SuiteCRM get troubled by varied results when they are sending emails.  I am of the opinion that this should be high-priority, because it has potential to greatly enhance the user experience.  Also I believe that not much coding is needed to achieve this functionality.

#### Your Environment

* SuiteCRM Version used: 7.7.7
* Browser name and version : Chrome Version 54.0.2840.99 m (64-bit)
* Environment name and version: Apache 2.2.25, PHP 5.4.19 & MySQL 5.5.33
* Operating System and version:  Linux/Windows

**Comment by shogunpol** [comment]
(tag and Close issue)

@LuminisIndia  - Thank you for highlighting this issue, however, this is a duplicate of issue #1366. This is issue will be closed, please follow the original issue.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Fix #1366 should be replicated for template parsing when sending emails via workflow](https://trello.com/c/7otcDfrh/51-fix-1366-should-be-replicated-for-template-parsing-when-sending-emails-via-workflow)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
